checker/src/main/java/org/checkerframework/checker/nullness/KeyForSubchecker.java

```diff
@@ -2,14 +2,37 @@
 
 import org.checkerframework.common.basetype.BaseTypeChecker;
 
+import java.util.NavigableSet;
+
 import javax.annotation.processing.SupportedOptions;
 
 /**
  * A type-checker for determining which values are keys for which maps. Typically used as part of
  * the compound checker for the nullness type system.
  *
+ * <p>You can use the following {@link SuppressWarnings} prefixes with this checker:
+ *
+ * <ul>
+ *   <li>{@code @SuppressWarnings("nullness")} suppresses warnings for nullness, initialization, and
```

Comment:
I see that this text comes from earlier text in the NullnessChecker. I think the `annotations` is misleading. It's not suppressing warnings based on what annotations are used, but on what checkers are run.
How about changing all these texts from "annotations" to e.g. "suppresses warnings from the Nullness, Initialization, and KeyFor Checkers"? That would also match the text in the manual and changelog.